pkg/config/remote/service/service.go

```diff
@@ -822,13 +822,15 @@ func (s *CoreAgentService) ClientGetConfigs(_ context.Context, request *pbgo.Cli
 		return nil, err
 	}
 
-	expires, err := s.uptane.TimestampExpires()
```

Comment:
can you add a comment on why these changes are done, these code flows can be tricky to read